tock/api/views.py

```diff
@@ -197,3 +200,115 @@ def bulk_timecard_list(request):
     queryset = get_timecards(TimecardList.queryset, request.GET)
     serializer = BulkTimecardSerializer()
     return stream_csv(queryset, serializer)
+
+
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+
+hours_by_quarter_query = '''
+with agg as (
+    select
+        extract(year from start_date) as year,
```

Comment:
Can't imagine doing this faster with the Django ORM. 
I think the year is slightly off for the gov calendar. [October, November, December](https://github.com/18F/tock/blob/master/tock/hours/models.py#L32) appear to be in the following year, otherwise this ready to merge.